notebooks/make_raw.py

```diff
@@ -22,7 +22,11 @@ def make_raw(
 
     # Extract annotations
     flashes = events.loc[events["label"] == "flash_begins"]
-    onsets = flashes.index.values.astype(np.int64) * 1e-9
+    mne_times = raw.times + recording_times[0]
+    signal_onsets = flashes.index.values.astype(np.int64) * 1e-9
+    mne_onsets = np.array(
+        [mne_times[np.argmin(abs(recording_times - onset))] for onset in signal_onsets]
+    )
```

Comment:
Nice catch! 💯 
So, if I understand correctly, the problem was  indeed that we had some drift in the signal that MNE raw object does not consider. 
🌱 We could send a warning if that drift is too important.